kalite/distributed/management/commands/kaserve.py

```diff
@@ -135,7 +135,10 @@ def handle(self, *args, **options):
         # them to be started up again as needed.
         Job.objects.update(is_running=False)
 
-        call_command("collectstatic", interactive=False)
+        # Copy static media, one reason for not symlinking: It is not cross-platform and can cause permission issues
+        # with many webservers
+        logging.info("Copying static media")
+        call_command("collectstatic", interactive=False, verbosity=0)
```

Comment:
Instead of silencing it for all cases, would it be better to just downgrade the logging level to DEBUG?